data/brands/shop/convenience.json

```diff
@@ -1270,6 +1270,18 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Dali",
+      "locationSet": {"include": ["ph"]},
+      "tags": {
+        "brand": "Dali",
+        "brand:website": "https://dali.ph",
```

Comment:
```suggestion
```

https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required
NSI doesn't collect urls, but you can find the P856 property in Wikidata and add the urls there